hack/configuration-anomaly-detection-template.Template.yaml

```diff
@@ -0,0 +1,136 @@
+apiVersion: v1
+kind: Template
+parameters: []
+metadata:
+  name: configuration-anomaly-detection-template
+objects:
+- apiVersion: v1
```

Comment:
You can leave that namespace in the template, but the pipeline will ignore it. Namespaces are managed by App-SRE